DOCSP-34836 Playgrounds Default DB in Connection String (#69)
mongodb/docs-mongodb-vscode: source/playgrounds.txt

an active connection, |vsce| displays
:guilabel:`Click here to add connection` at the top of your playground.

.. procedure::
   :style: normal
   
   .. step:: Select :guilabel:`Click here to add connection` 

>       After you click :guilabel:`Click here to add connection`, |vsce| opens the 
>       connection string dialog.

Reviewer: ```suggestion
      After you click :guilabel:`Click here to add connection`, the
      connection string dialog displays.
```

Author: Opting to find a middle ground here since ending the sentence with "display" makes it sound like there's a missing a direct object.

      .. figure:: /images/vsce-playground-add-connection-btn.png
         :figwidth: 700px
         :alt: Link to add connection from playground
         

>    .. step:: Enter your connection string

Reviewer: I had to click on extra time before the connection string dialog appeared:

<img width="615" alt="Screenshot 2023-12-08 at 5 34 38 PM" src="https://github.com/mongodb/docs-mongodb-vscode/assets/85948430/bde3d254-177d-443e-a436-bf27f0d214c6">

Author: Added an additional step about this + included information on how previously connected deployments will show up as options.

>       Enter the :manual:`connection string </reference/connection-string/>` for 

Reviewer: **Suggestion**

It may be valuable to enter a small sample connection string here to avoid the users needing to go to the additional page WDYT?
Something like:

```
mongodb://localhost:27017/mydatabase
```

Author: I'm leaning towards keeping this as is because I'm expecting that a user coming to use the VSCode extension playground would be familiar with MDB basics such as using a connection string, given they must already have a deployment set up.

>       Enter the :manual:`connection string </reference/connection-string/>` for 
>       the deployment you want to run this playground against.

Reviewer: ```suggestion
      Enter a :manual:`connection string </reference/connection-string/>` to connect to your deployment.
```

      .. note:: 
         

>          If the connection string specifies a database, the playground runs 

Reviewer: **suggestion**

Should this be does not?

```suggestion
         If the connection string does not specify a database, the playground runs 
```

Author: Ah I might've worded it weirdly. the behavior is tht whatever database is specified in the connection string **is** the default database.

>          against the default connected database unless you explicitly call 
>          ``use('<database_name>')``  to use another database.

Reviewer: ```suggestion
         against the default connected database unless you explicitly switch databases with 
         ``use('<database_name>')``.
```

Author: Split this into two sentences. LMK what you think.

         If you are connected to a default database, the playground 

>          autocompletes only for collection names available on the default 
>          database. 

Reviewer: ```suggestion
         autocompletes only for collection names available on that database. 
```

         
   .. step:: Press :guilabel:`Enter`

      After you enter your connection string, a confirmation message replaces 
      the :guilabel:`Click here to add connection` link. The confirmation 
      message includes information on the connected deployment and, if 

>       applicable, the default database. 

Reviewer: I think you can shorten this to:

The confirmation message includes information on the connected deployment.

I dont see anything siginicant on the default database information, let me know if I am looking at the wrong spot!:

 
<img width="482" alt="Screenshot 2023-12-08 at 5 40 08 PM" src="https://github.com/mongodb/docs-mongodb-vscode/assets/85948430/4fa5a5d3-a3a2-45d8-a657-784f39679d1c">

Author: The original ticket filer wanted us to make a note about this because multiple users have shared feedback that weren't aware they were in the incorrect database, which prevented them from connecting to collections in other dbs. 

<img width="792" alt="connection-with-default-connected-db" src="https://github.com/mongodb/docs-mongodb-vscode/assets/112565127/59290cd7-ab7d-488a-bbe1-3e11b6f3f704">

Author: I can change the wording to say "current" database rather than default.

      After you click :guilabel:`Click here to add connection`, |vsce| displays 
      the connection string drop-down menu.

      .. figure:: /images/vsce-playground-add-connection-btn.png
         :figwidth: 700px
         :alt: Link to add connection from playground

   .. step:: Click :guilabel:`Add new connection`

      In the connection string drop-down menu, select 
      :guilabel:`Add new connection` to enter a new connection string. If you 
      have previously connected to the playground with another deployment, the 
      connection string will appear as an option under the drop-down menu. 

   .. step:: Enter your connection string
      
      Enter the :manual:`connection string </reference/connection-string/>` to 
      connect to your deployment.

      .. note:: 
         
         If the connection string specifies a database the playground runs 
         against that database by default. To switch databases,   
         call ``use('<database_name>')``.

         If you are connected to a default database, the playground 
         autocompletes only for collection names available on that 
         database. 
         
   .. step:: Press :guilabel:`Enter`

>       After you enter your connection string, a confirmation message replaces 

Reviewer 2: Nit, fine as is, in VSCode these `confirmation` messages are called `code lenses`. https://code.visualstudio.com/blogs/2017/02/12/code-lens-roundup

Author: Gotcha, thanks! Will update.
